backend/Tasque.BLL/Services/EntityCrudService.cs

```diff
@@ -0,0 +1,49 @@
+﻿using Tasque.Core.Common.Entities.Abstract;
+using Tasque.Core.DAL;
+
+namespace Tasque.Core.BLL.Services
+{
+    public abstract class EntityCrudService<TEntity> : IEntityCrudService<TEntity> where TEntity : BaseEntity
+    {
+        private readonly DataContext _db;
+        public EntityCrudService(DataContext db)
+        {
+            _db = db;
+        }
+
+        public virtual TEntity Create(TEntity entity)
+        {
+            _db.Set<TEntity>().Add(entity);
+            _db.SaveChanges();
```

Comment:
It would just make the overall performance worse.

https://docs.microsoft.com/en-us/dotnet/api/microsoft.entityframeworkcore.dbset-1.addasync?view=efcore-6.0#definition

Reply:
not add but save